ConsoleAppn7/ConsoleAppn7/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите X");
+        var x = int.Parse(Console.ReadLine());
+        Console.WriteLine($"полученное значение функции:" + Coordinate(x));
+        Console.ReadLine();
+    }
+    static double Coordinate(int x)
```

Comment:
Неудачное имя метода. Причем тут координата? Уж лучше так:
```suggestion
        Console.WriteLine($"полученное значение функции:" + MyFunction(x));
        Console.ReadLine();
    }
    static double MyFunction(int x)
```